templates/internal/appName/k8s/activity.go.tpl

```diff
@@ -1,5 +1,5 @@
 // {{ stencil.ApplyTemplate "copyright" }}
-{{- $_ := file.SetPath (printf "internal/%s/kubernetes.go" .Config.Name) }}
+  {{- $_ := file.SetPath (printf "internal/%s/kubernetes.go" .Config.Name) }}
```

Comment:
Nit, extra space here?

Reply:
huh, that's weird.  the {{- right after it will nullify it, but prolly should fix that in master

Reply:
https://github.com/getoutreach/stencil-golang/pull/204